api/api_gateway.go

```diff
@@ -34,6 +35,12 @@ import (
 //  * Generate openrpc blobs
 
 type Gateway interface {
+	ChainGetNode(ctx context.Context, p string) (*IpldObject, error)
```

Comment:
why this one @dumikau ?

Reply:
tbh I can't see this one making it to the v2 API that's under development now without a really good use-case, are people asking for it?

Reply:
Yes, this method was added by request of a user from Neti-Soft